backend/src/services/course/course.schemas.js

```diff
@@ -218,9 +218,14 @@ export const AddUserInCourseSchema = {
   body: {
     type: 'object',
     properties: {
-      user_id: { type: 'number' },
+      email: { type: 'string', format: 'email' },
+      role: {
+        type: 'string',
+        enum: ['student', 'ta', 'professor'],
+        default: 'student',
+      },
```

Comment:
[Suggestion]
We could use our enum definitions in code directly here as well.
I didn't know we could define enum response types too. This is great!